primer.md

```diff
@@ -145,6 +149,13 @@ user-stories to explain the rationale and need for them. This supporting
 information will be added to the [Prior Art](#prior-art) section of this
 document.
 
+Extension attributes to the CloudEvent specification are meant to
+be additional metadata that needs to be included to help ensure proper
+routing and processing of the CloudEvent. Additional metadata for other
+purposes, that is related to the event itself and not needed in the
```

Comment:
1. Again, why do we restrict the metadata purpose to be just "transportation of the CloudEvent"? 
2. Could you clarify what is meant by  "extensibility points of the event itself."?

Reply:
I added some text to try to clear it up.